### PR TITLE
Update Fresh Egg Australia Listing

### PR DESCRIPTION
**Pull request by kellyfinn**
Listing included UK info not Australia info.

### CODE REVIEW

**Comment by kellyfinn** [comment]
@stewsmith Can you take a look at this for me? I updated Fresh Egg Australia listing which had Fresh Egg UK info listed (I know at one point it was accurate...) but now it isn't appearing in the preview.

Thanks for the help!

File: website/partners/solutions/freshegg-australia/index.hbs [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
-    state: "United Kingdom"
-    address1: "1-13 Buckingham Road"
-    address2: ""
-    zip: "BN11 1TH"
-    country: "UK"
+    city: "Sydney"
+    state: "NSW"
+    address1: "101 William Street, Darlinghurst"
+    address2: "Level 2"
+    zip: "2010
```
Comment (stewsmith) [inline website/partners/solutions/freshegg-australia/index.hbs]: Here's the problem. I'll fix now.

**Comment by stewsmith** [comment]
@kellyfinn problem was caused by an unclosed quote. Those are a typical culprit when you don't see your changes.

**Comment by stewsmith** [comment]
LGTM

**Pull request merged**